library/psa_crypto_ecp.c

```diff
@@ -74,10 +74,18 @@ psa_status_t mbedtls_psa_ecp_load_representation(
     }
 
 #if defined (MBEDTLS_PSA_CRYPTO_ACCEL_DRV_C) && defined(MBEDTLS_ECP_ALT)
+    if (PSA_KEY_TYPE_IS_VENDOR_DEFINED(type))
+    {
     curve_bytes = ( 0U != PSA_ECC_BYTES_VENDOR_RAW(curve_bytes) ) ?
                     PSA_ECC_BYTES_VENDOR_RAW(curve_bytes) : curve_bytes ;
 
     curve_bits = PSA_BYTES_TO_BITS(curve_bytes);
+
+    if((curve_bits == 528U) && (curve_bytes == 66U))
```

Comment:
Isnt it possible to build this logic into the macros above?

Reply:
Here curve_bytes= 66 and curve bits = curve_bytes * 8 = 528, this calculation works fine for 256 and 384 as they are aligned and 521 is not. I mean those 2 are divisible by 8. 
grp_id will be expecting 521, but in previous calculation we get it as 528.